content/blog/2018-03-20-door.md

```diff
@@ -0,0 +1,179 @@
+---
+slug: "door"
+title: The Database of Odorant Responses
+package_version: 2.0.1
+authors:
+  - name: Daniel Münch
+    url: https://github.com/dahaniel
+date: 2018-03-20
+categories: blog
+topicid:
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- package
+- Drosophila
+- odor responses
+- physiology
+- neuroscience
+- data-only-pkg
+- visualization
+- modelling
+---
+
+# DoOR - The Database of Odorant Responses
+## Olfactory Coding
```

Comment:
The headings fonts look very big in preview here https://deploy-preview-158--ropensci.netlify.com/blog/2018/03/20/door/. Please try ## or ###